docs/dev-guide/building-docs.md

```diff
@@ -49,12 +49,12 @@ The documentation follows the [Diátaxis](https://diataxis.fr/) framework with f
    - Getting started guides
    - Learning-oriented content
 
-2. **Technical Reference** (`docs/reference/`)
+2. **Technical References** (`docs/references/`)
```

Comment:
Up to you, but it is more standard for the URL to be `/reference/XXX` or `/explanation/XXX`:

https://docs.pytest.org/en/stable/reference/fixtures.html

https://docs.pytest.org/en/stable/how-to/assert.html

etc.